README.md

```diff
@@ -1 +1,1146 @@
+
+
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
+
+### Optional Requirements
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina), [Atom](https://atom.io/packages/language-ballerina))
+- [Docker](https://docs.docker.com/engine/installation/)
+- [Kubernetes](https://kubernetes.io/docs/setup/)
+
+## Implementation
+
+### Create the project structure
+
+Ballerina is a complete programming language that supports custom project structures. Use the following package structure for this guide.
+
+```
+ └── guide
+     ├── message_transformation
+     │   └── message_transformation.bal
+     │  
+     └── tests
+         └── message_transformation_test.bal
+
+```
+
+Create the above directories in your local machine and also create empty `.bal` files.
+
+Open the terminal and navigate to `# eip-message-transformation/guide` and run the Ballerina project initializing toolkit.
+
+```bash
+   $ ballerina init
+```
+### Developing the SQL data
+
+Ballerina language has built-in support for writing web services. The `service` keyword in Ballerina simply defines a web service. Inside the service block, we can have all the required resources. You can define a resource inside the service. You can implement the business logic inside a resource using Ballerina language syntax.
+We can use the schema of the following database to store the student's data and student's results data.
+
+tsetdb database used to store student's data in student table as below.
+``` 
++---------+--------------+------+-----+---------+-------+
+| Field   | Type         | Null | Key | Default | Extra |
++---------+--------------+------+-----+---------+-------+
+| id      | int(11)      | NO   | PRI | NULL    |       |
+| name    | varchar(255) | YES  |     | NULL    |       |
+| address | varchar(255) | YES  |     | NULL    |       |
+| gender  | varchar(255) | YES  |     | NULL    |       |
++---------+--------------+------+-----+---------+-------+
+```
+tsetdb1 database used to store student's results data in StudentDetails table as below.
+
+``` 
++-----------+------------+------+-----+---------+-------+
+| Field     | Type       | Null | Key | Default | Extra |
++-----------+------------+------+-----+---------+-------+
+| ID        | int(11)    | NO   | PRI | NULL    |       |
+| Com_Maths | varchar(1) | YES  |     | NULL    |       |
+| Physics   | varchar(1) | YES  |     | NULL    |       |
+| Chemistry | varchar(1) | YES  |     | NULL    |       |
++-----------+------------+------+-----+---------+-------+
+```
+In the below service. it added the student's data through the request. But student's results data must be located in the database as below.
```

Comment:
Change to 'The student's data is added through the request in the following service. However, the student's result data must be located in the database as shown below.'

Reply:
Done